package.json

```diff
@@ -63,10 +64,12 @@
     "okapiInterfaces": {
       "acquisitions-units": "1.1",
       "configuration": "2.0",
-      "invoice": "3.0",
+      "invoice": "3.1",
+      "finance.funds": "1.0",
       "orders": "8.0",
-      "organizations-storage.organizations": "2.1",
-      "users": "15.0",
+      "organizations-storage.organizations": "2.2",
```

Comment:
usually we don't update minor versions of interfaces. Even if you want to do this - it's better to specify a previous version as well, to not break the build process in case there is still old version is deployed to the env